src/UI/Button.elm

```diff
@@ -451,7 +454,7 @@ elFromBody : RenderConfig -> ButtonBody msg -> Element msg
 elFromBody cfg body =
     case body of
         BodyText str ->
-            Text.body1 str
+            Text.subtitle2 str
```

Comment:
If "Text attributes are the same of subtitle1" why are you invoking `subtitle2`?